py3-cassandra-medusa.yaml

```diff
@@ -48,35 +45,34 @@ pipeline:
       POETRY_VIRTUALENVS_IN_PROJECT=true poetry install
       poetry build
 
-  - runs: |
       # Setup the virtualenv
-      python${{vars.py-version}} -m venv .venv --system-site-packages
+      python${{vars.py-version}} -m venv .venv
```

Comment:
I've reviewed and can't think of any case we might need '--system-site-packages' for this package ,but checking this change was intentional cleanup?

Reply:
@mamccorm it's intentional. We install poetry above as a precursor to building medusa. If we enable site packages, Python picks up on Poetry and doesn't install it inside the virtual environment when we reinstall below. It's just a difference in the config file, and we enable system site packages after installing Poetry in the virtual environment